<launch>
    <!-- vim: set ft=xml noet : -->

    <arg name="transform_json_path" default="/home/orangepi/robocup_ws/src/conductor/example/json/transforms.json" />
    <arg name="waypoint_json_path" default="/home/orangepi/robocup_ws/src/conductor/example/json/rectangle_waypoint.json" />
    <arg name="log_output" default="screen" />
    <arg name="respawn_conductor" default="false" />
    <arg name="startup_delay" default="10" />

    <include file="$(find conductor)/launch/robocup.launch">
        <arg name="transform_json_path" value="$(arg transform_json_path)" />
        <arg name="waypoint_json_path" value="$(arg waypoint_json_path)" />
        <arg name="log_output" value="$(arg log_output)" />
        <arg name="respawn_conductor" value="$(arg respawn_conductor)" />
        <arg name="startup_delay" value="$(arg startup_delay)" />
    </include>

    <include file="$(find mavros)/launch/apm.launch">
    </include>

    <include file="$(find livox_ros_driver2)/launch_ROS1/msg_MID360.launch">
    </include>

    <include file="$(find faster_lio)/launch/mapping_mid360.launch">
    </include>

    <include file="$(find vision_to_mavros)/launch/mid360_to_mavros.launch">
    </include>
</launch>